lib/shared_data/include/SharedDataTypes.h

```diff
@@ -49,12 +50,15 @@ struct DrivetrainDynamicReport_s
     float measuredTorqueCurrents[NUM_MOTORS];
     float measuredMagnetizingCurrents[NUM_MOTORS];
 };
+/// @brief stores two arrays of size equal to the amount of motors: speed setpoints and necessary torque to reach those setpoints 
 struct DrivetrainCommand_s
 {
     float speeds_rpm[NUM_MOTORS];
     float torqueSetpoints[NUM_MOTORS]; // FIXME: misnomer. This represents the magnitude of the torque the inverter can command to reach the commanded speed setpoint
 };
 
+/// @brief Packages drivebrain command with ready boolean
```

Comment:
drivetrain not drivebrain, more than just the drivebrain can command the drivetrain